src/pages/workspace/accounting/qbd/export/QuickbooksDesktopExportPage.tsx

```diff
@@ -0,0 +1,121 @@
+            title: qbdConfig?.exportDate ? translate(`workspace.qbo.exportDate.values.${qbdConfig?.exportDate}.label`) : undefined,
+            subscribedSettings: [CONST.QUICKBOOKS_CONFIG.EXPORT_DATE],
+        },
+        {
+            description: translate('workspace.accounting.exportOutOfPocket'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_OUT_OF_POCKET_EXPENSES.getRoute(policyID)),
+            title: qbdConfig?.reimbursableExpensesExportDestination ? translate(`workspace.qbo.accounts.${qbdConfig?.reimbursableExpensesExportDestination}`) : undefined, // TODO: should be updated to use new routes
+            subscribedSettings: [CONST.QUICKBOOKS_CONFIG.REIMBURSABLE_EXPENSES_EXPORT_DESTINATION, CONST.QUICKBOOKS_CONFIG.REIMBURSABLE_EXPENSES_ACCOUNT],
+        },
+        {
+            description: translate('workspace.qbo.exportInvoices'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_INVOICE_ACCOUNT_SELECT.getRoute(policyID)), // TODO: should be updated to use new routes
+            title: qbdConfig?.receivableAccount?.name,
+            subscribedSettings: [CONST.QUICKBOOKS_CONFIG.RECEIVABLE_ACCOUNT],
+        },
+        {
+            description: translate('workspace.accounting.exportCompanyCard'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_ACCOUNT.getRoute(policyID)), // TODO: should be updated to use new routes
+            brickRoadIndicator: qbdConfig?.errorFields?.exportCompanyCard ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: qbdConfig?.nonReimbursableExpensesExportDestination ? translate(`workspace.qbo.accounts.${qbdConfig?.nonReimbursableExpensesExportDestination}`) : undefined,
+            subscribedSettings: [
+                CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_EXPENSES_EXPORT_DESTINATION,
+                CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_EXPENSE_ACCOUNT,
+                ...(shouldShowVendorMenuItems ? [CONST.QUICKBOOKS_CONFIG.AUTO_CREATE_VENDOR] : []),
+                ...(shouldShowVendorMenuItems && qbdConfig?.autoCreateVendor ? [CONST.QUICKBOOKS_CONFIG.NON_REIMBURSABLE_BILL_DEFAULT_VENDOR] : []),
+            ],
+        },
+        {
+            description: translate('workspace.qbo.exportExpensifyCard'),
+            title: translate('workspace.qbo.accounts.credit_card'),
+            shouldShowRightIcon: false,
+            interactive: false,
+        },
+    ];
+
+    const accessVariants = canUseNewDotQBD ? [] : [CONST.POLICY.ACCESS_VARIANTS.ADMIN];
+
+    return (
+        <ConnectionLayout
+            displayName={QuickbooksDesktopExportPage.displayName}
+            headerTitle="workspace.accounting.export"
+            title="workspace.qbd.exportDescription"
+            accessVariants={accessVariants}
```

Comment:
What I meant for using the QBD beta for access variants, is we should only allow access for now if the user is on the QBD beta. 
```suggestion
            accessVariants={canUseNewDotQBD} // TODO: remove it once the QBD beta is done
```

Otherwise right now if the user is an admin, he would be able to access a page that is just not ready yet. 
It would confuse him and degrade his experience.

Reply:
Thank you!